snippets/csharp/VS_Snippets_CLR/RegularExpressions.Examples.HREF/cs/example.cs

```diff
@@ -4,45 +4,47 @@
 [assembly: CLSCompliant(true)]
 public class Class1
 {
-   // <Snippet2>
-   public static void Main()
-   {
-      string inputString = "My favorite web sites include:</P>" +
-                           "<A HREF=\"http://msdn2.microsoft.com\">" +
-                           "MSDN Home Page</A></P>" +
-                           "<A HREF=\"http://www.microsoft.com\">" +
-                           "Microsoft Corporation Home Page</A></P>" +
-                           "<A HREF=\"http://blogs.msdn.com/bclteam\">" +
-                           ".NET Base Class Library blog</A></P>";
-      DumpHRefs(inputString);                     
+    // <Snippet2>
+    public static void Main()
+    {
+        string inputString = "My favorite web sites include:</P>" +
+                             "<A HREF=\"http://msdn2.microsoft.com\">" +
+                             "MSDN Home Page</A></P>" +
+                             "<A HREF=\"http://www.microsoft.com\">" +
+                             "Microsoft Corporation Home Page</A></P>" +
+                             "<A HREF=\"http://blogs.msdn.com/bclteam\">" +
+                             ".NET Base Class Library blog</A></P>";
+        DumpHRefs(inputString);
 
-   }
-   // The example displays the following output:
-   //       Found href http://msdn2.microsoft.com at 43
-   //       Found href http://www.microsoft.com at 102
-   //       Found href http://blogs.msdn.com/bclteam at 176
-   // </Snippet2>
+    }
+    // The example displays the following output:
+    //       Found href http://msdn2.microsoft.com at 43
+    //       Found href http://www.microsoft.com at 102
+    //       Found href http://blogs.msdn.com/bclteam at 176
+    // </Snippet2>
    
-   // <Snippet1>
-   private static void DumpHRefs(string inputString) 
-   {
-      Match m;
-      string HRefPattern = "href\\s*=\\s*(?:[\"'](?<1>[^\"']*)[\"']|(?<1>\\S+))";
+    // <Snippet1>
+    private static void DumpHRefs(string inputString) 
+    {
+        Match m;
+        string HRefPattern = @"href\s*=\s*(?:[""'](?<1>[^""']*)[""']|(?<1>\S+))";
```

Comment:
@mairaw What's your opinion here?  Is the `""` from the `@` string better than `\"` or not? I'm not sure.

Reply:
I still prefer the verbatim better. The reason why I created the issue was because there were several mistakes in the docs/code for that area and it was hard to compare text and code patterns without that. Knowing it's just the quote that will be different makes it easier for me to review those.